arbos/addressTable/addressTable.go

```diff
@@ -25,7 +25,7 @@ func Initialize(sto *storage.Storage) {
 
```

Comment:
let's rename (or if you don't want this to have conflicts with execution split just add TODO for it so that we track it) package name from `addressTable` to `addresstable` since package names should be lowercase [0].

[0] https://google.github.io/styleguide/go/decisions#package-names